assets/src/edit-story/components/panels/design/imageAccessibility/imageAccessibility.js

```diff
@@ -59,20 +55,21 @@ function ImageAccessibilityPanel({ selectedElements, pushUpdate }) {
       isPersistable={!highlight}
```

Comment:
@barklund Do you know why is the `highlight` related to `isPersistable` (in case you remember where this came from)? Not completely sure what's happening with it.

Reply:
Removing this then, seems to be style-related.